starter-code/src/components/Contacts.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
+import Contacts from "/src/contacts.json";
+import styled from "styled-components";
+
+const Tablestyle = styled.div`
+  display: flex;
+  justify-content: center;
+  img {
+    width: 100%;
+  }
+  th {
+    word-spacing: 100vw;
+    width: 5vw;
+  }
+`;
+
+const Button = styled.button`
+  background-color: #4caf50;
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+`;
+
+const Celebrity = ({ picture, name, popularity, index, deleteFunction }) => {
+  return (
+    <tr>
+      <th>
+        <img src={picture} alt={name} />
+      </th>
+      <th>{name}</th>
+      <th>{popularity.toFixed(2)}</th>
+      <button onClick={() => deleteFunction(index)}>Delete</button>
```

Comment:
Fijate lo que te da 

Warning: validateDOMNesting(...): <button> cannot appear as a child of <tr>.